src/fsharp/FSharp.Core/map.fsi

```diff
@@ -85,6 +85,9 @@ namespace Microsoft.FSharp.Collections
         /// <returns><c>true</c> if the value is present, <c>false</c> if not.</returns>
         member TryGetValue: key:'Key * [<System.Runtime.InteropServices.Out>] value:byref<'Value> -> bool
 
+        /// <summary>The keys in the map.</summary>
```

Comment:
Please add property `.Values`

Reply:
I saw your comment from earlier:

> @dsyme I found out that the Map.Values implementation collides with internal Map.Values extension in illib.fs - MapAutoOpens module which returns a list instead of seq. This is the reason the solution right now doesn't build.

Yes the one in `illib.fs` should be renamed or removed as part of this PR.

However I have some concerns about Keys and Values, I'll discuss on the suggestion thread

Reply:
Ok, I'll remove the illib.fs implementation and fix all the usages as part of this PR. I'll keep it as seq<'Key> for now but I'm happy to update when decided.

Reply:
Great!